src/main/kotlin/no/nav/familie/klage/brev/baks/BaksBrevController.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.familie.klage.brev.baks
+
+import no.nav.familie.klage.felles.domain.AuditLoggerEvent
+import no.nav.familie.klage.infrastruktur.sikkerhet.TilgangService
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.UUID
+
+@RestController
+@RequestMapping(path = ["/api/baks/brev"])
+@ProtectedWithClaims(issuer = "azuread")
+@Validated
+class BaksBrevController(
+    private val baksBrevService: BaksBrevService,
+    private val tilgangService: TilgangService,
+) {
+    @GetMapping("/{behandlingId}/pdf")
+    fun hentBrev(
```

Comment:
Kanskje renames til noe som hentBrevPdf siden man ikke henter domene objektet brev her, men pdf innholdet.

POST /api/baks/brev/{behandlingId} for opprettelse av brev
GET /api/baks/brev/{behandlingId}/pdf for henting av brev i pdf format